backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/webservice/schema/CardQueryService.kt

```diff
@@ -8,7 +9,7 @@ import org.postgresql.util.Base64
 @Suppress("unused")
 class CardQueryService {
     @GraphQLDescription("Returns whether there is a card with that hash registered for that this TOTP is currently valid")
-    fun verifyCard(card: CardVerificationModel): Boolean {
-        return CardVerifier.verifyCardHash(Base64.decode(card.cardDetailsHashBase64), card.totp)
+    fun verifyCard(project: String = DEFAULT_PROJECT, card: CardVerificationModel): Boolean {
```

Comment:
```suggestion
    fun verifyCard(project: String, card: CardVerificationModel): Boolean {
```

I think we can also break this endpoint.